### PR TITLE
String implementation

### PR DESCRIPTION
**Pull request by arccth**

### CODE REVIEW

File: course4/homework1/vector2d.h [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
 
 struct Vector2d
 {
-	// TODO
+private:
+   double mX, mY;
+
+public:
+
+   Vector2d(const double x, const double y):
```
Comment (maciekgajewski) [inline course4/homework1/vector2d.h:13]: Please, deliver the string homework in new pull request.

**Review by maciekgajewski — COMMENTED** [review]
Please, use separate pull request for new homework.

File: course4/homework1/vector2d.h (hunk 1 above)
Comment (arccth) [reply course4/homework1/vector2d.h:13]: But it is separate, just branched from homework 1. First pull request here: https://github.com/maciekgajewski/c-excercises/pull/51

Comment (maciekgajewski) [reply course4/homework1/vector2d.h:13]: It is not separate. It is not a string homework submission if there are vector changes in the patch.

**Pull request closed without merging**